src/util/http_utils.js

```diff
@@ -3,7 +3,8 @@
 /* eslint-disable no-control-regex */
 
 const _ = require('lodash');
-const ip = require('ip');
+const ip_module = require('ip');
```

Comment:
why are we just changing the name here? consistency? maybe explain here that this change is still in progress, and we plan to remove this module. 
Is the effort of totally removing this module that big? the isEqual sounds easy also, CIDR doesn't suppose to be hard if we have the ip_to_long conversion

Reply:
This is Phase 1, as the title suggests.
The change of this name is to make it easy for us to look for `ip_module.` 
in later PR we will address the other functions, CIDR is not that easy...

Reply:
the question is if we need 2 phases :). But sure, if you prefer to split.